tests/Unit/Client/UpdaterTest.php

```diff
@@ -168,4 +171,30 @@ public function testCheckFreezeAttackAttack() : void
         // be thrown.
         $method->invoke($sut, $signedMetadata, $now);
     }
+
+    /**
+     * Test that TUF will transparently verify downloaded target hashes.
```

Comment:
I know this isn't drupal core but nit `Tests` 😂

Reply:
Fixed.